ab_tool/views/intervention_point_pages.py

```diff
@@ -78,21 +79,22 @@ def submit_create_intervention_point(request, experiment_id):
     """ Note: request will always be POST because Canvas fetches pages within iframe by POST
         TODO: use Django forms library to save instead of getting individual POST params """
     course_id = get_lti_param(request, "custom_canvas_course_id")
-    name = post_param(request, "name")
+    name = validate_name(post_param(request, "name"))
     notes = post_param(request, "notes")
+    intervention_pointurls = [(k,v) for (k,v) in request.POST.iteritems()
+                              if STAGE_URL_TAG in k and v]
+    for (k,v) in intervention_pointurls:
```

Comment:
Could you integrate the validation call into the list comprehension above instead of iterating again here?